### PR TITLE
Remove 14.0.1+7.1 and 15.0.0+24.1 nightlies from the api

### PR DESCRIPTION
**Pull request by johnoliver**
# Checklist

- [ ] You added tests to cover the change
- [ ] `mvn clean install` build and test completes
- [ ] You changed or added to the documentation

### CODE REVIEW

**Comment by gdams** [comment]
@johnoliver what's the reason behind this?

**Comment by johnoliver** [comment]
@gdams to fix #213

**Comment by sxa** [comment]
@gdams See discussion in #api but it's a fix for https://github.com/AdoptOpenJDK/openjdk-api-v3/issues/213

File: adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/mapping/adopt/AdoptReleaseMapper.kt [hunk 1]
```diff
@@ -68,12 +68,23 @@ object AdoptReleaseMapper : ReleaseMapper() {
...
+                .filter { release -> !excludeRelease(release) }
         } catch (e: FailedToParse) {
             LOGGER.error("Failed to parse $releaseName")
             throw e
         }
     }
 
+    private fun excludeRelease(release: Release): Boolean {
+        return if (release.release_type == ReleaseType.ea) {
+            // remove all 14.0.1+7.1 and 15.0.0+24.1 nightlies - https://github.com/AdoptOpenJDK/openjdk-api-v3/issues/213
```
Comment (karianna) [inline adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/mapping/adopt/AdoptReleaseMapper.kt:80]: We should probably externalize this into a properties file.

**Pull request merged**